backend/controller/leases/dal/dal.go~

```diff
@@ -0,0 +1,18 @@
+// Package dal provides a data abstraction layer for leases
```

Comment:
Delete this file

Reply:
d'oh - I forgot to push my last commit. Fixed